src/HotChocolate/Core/src/Validation/DocumentValidatorVisitor.cs

```diff
@@ -0,0 +1,71 @@
+using System.Collections.Generic;
+using System.Linq;
+using HotChocolate.Language;
+using HotChocolate.Language.Visitors;
+
+namespace HotChocolate.Validation
+{
+    public class DocumentValidatorVisitor
+        : SyntaxWalker<IDocumentValidatorContext>
+    {
+        protected DocumentValidatorVisitor()
+            : base(Continue)
```

Comment:
I think `Continue`is default

Reply:
Skip is I think

Reply:
https://github.com/ChilliCream/hotchocolate/pull/1601/files/8d7ada9c3f21ae6574b9a0778da1c221892d89b3#diff-9bbbb08c02ef8a27c92aa7aa0d9c0817R7

Reply:
public SyntaxVisitor()
         {
             DefaultAction = Skip;
         }

Reply:
If not specified it is skip